cli/tests/program.rs

```diff
@@ -304,8 +310,15 @@ fn test_cli_program_deploy_no_authority() {
         .start_with_mint_address(mint_pubkey, SocketAddrSpace::Unspecified)
         .expect("validator start failed");
 
-    let rpc_client =
-        RpcClient::new_with_commitment(test_validator.rpc_url(), CommitmentConfig::processed());
+    let mut config = CliConfig::recent_for_tests();
```

Comment:
Maybe be we can create a helper function to encapsulate this duplicated code?

Reply:
Certainly, done!